### DISCUSSION TITLE
Forge keeps cloning itself into "stable-diffusion-webui" inside of it's own install folder.

### DISCUSSION THREAD

**Discussion by KillavoltAI** [lllyasviel/stable-diffusion-webui-forge#506] [General]
IT seems that when I change a setting in webui-user.sh and then change it back or comment the line again it just keeps using the changes...or I've done something wrong because now when I run webui.sh ForgeUI clones itself into another folder within it's install folder and now I can't get it to stop doing that again.

**Comment by YordanVelkov** [comment]
Hi,
I'm not sure if this could help but try :
bash webui.sh -f --skip-version-check --skip-install

or if you like more args:  
bash webui.sh -f --skip-version-check --skip-install --api --xformers

-f is to run webui.sh from root user
- the other args should be self-explanatory 

I hope this helps
